### PR TITLE
gracefully handle empty input

### PR DESCRIPTION
**Pull request by winglian**

### CODE REVIEW

**Comment by winglian** [comment]
fixes #438

File: src/axolotl/prompt_tokenizers.py [hunk 1]
```diff
@@ -85,7 +85,11 @@ def _tokenize(self, prompt: str, add_eos_token=True, strip_bos_token=False):
             result["input_ids"].append(self.tokenizer.eos_token_id)
             result["attention_mask"].append(1)
 
-        if result["input_ids"][0] == self.tokenizer.bos_token_id and strip_bos_token:
+        if (
+            len(result["input_ids"]) > 0
```
Comment (NanoCode012) [inline src/axolotl/prompt_tokenizers.py:89]: Could we also add a LOG.info / warn if length is 0? Same like a previous PR?

Comment (winglian) [reply src/axolotl/prompt_tokenizers.py:89]: we already warn about in line 78

Comment (NanoCode012) [inline src/axolotl/prompt_tokenizers.py:89]: I guess if we wanted to reuse variables, we can optionally extract this to a variable and reuse the variable instead. This PR is good enough though.

**Review by NanoCode012 — APPROVED** [review]

**Pull request merged**